lib/attribute.js

```diff
@@ -445,6 +445,7 @@ validators.addFormat = function addFormat(name, handler) {
  *   - alpha
  *   - alpha-numeric
  *   - utc-millisec
+ *   - number
```

Comment:
AFAIK, this isn't a valid value for `format`, and we shouldn't list our custom formats here anyway, because they won't be upstreamed